lib/decidim/reporting_proposals/engine.rb

```diff
@@ -14,10 +14,13 @@ class Engine < ::Rails::Engine
       initializer "decidim_reporting_proposals.overrides", after: "decidim.action_controller" do
         config.to_prepare do
           Decidim::Admin::ComponentsController.include(Decidim::ReportingProposals::Admin::NeedsHeaderSnippets)
+          Decidim::Proposals::Admin::ProposalsController.include(Decidim::ReportingProposals::Admin::ProposalsControllerOverride)
```

Comment:
```suggestion
```
We won't be needing this override if we user our own controller for the `hide` action